R/plotly.R

```diff
@@ -254,48 +254,64 @@ plotly_build <- function(l = last_plot()) {
   # this is ugly, but I think it is necessary, since we don't know how many 
   # traces we have until we evaluate args and call traceify() (or similar)
   x <- list()
-  for (i in seq_along(l$data)) {
-    d <- l$data[[i]]
+  x$data <- unlist(lapply(l$data, function(d) {
```

Comment:
@cpsievert Yes, the function inside `lapply()` returns a list of traces for each `l$data` element. `unlist()` just combines these lists into one.